### PR TITLE
test: adding advisory test cases and decorators

### PR DESCRIPTION
**Pull request by anthraxx**

### CODE REVIEW

File: app/advisory.py [hunk 1]
```diff
@@ -80,3 +80,13 @@ def advisory_extend_model_from_advisory_text(advisory):
     advisory.impact = advisory_get_impact_from_text(advisory.content)
     advisory.workaround = advisory_get_workaround_from_text(advisory.content)
     return advisory
+
+
+def advisory_get_date_label(utctimetuple=None):
+    now = utctimetuple if utctimetuple else datetime.utcnow().utctimetuple()
+    return '{}{}'.format(now.tm_year, '{}'.format(now.tm_mon).rjust(2, '0'))
```
Comment (SantiagoTorres) [inline app/advisory.py]: I wonder if we'd rather use
```python
return '{}{:02}'.format(now.tm_year, now.tm_mon)
```
for the sake of clarity.

File: test/test_advisory.py [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+    assert workaround == advisory.workaround
+    assert impact == advisory.impact
+    assert reference == advisory.reference
+
+
+@create_package(name='foo', version='1.2.3-4')
+@create_group(id=DEFAULT_GROUP_ID, issues=[DEFAULT_ISSUE_ID], packages=['foo'], affected='1.2.3-3', fixed='1.2.3-4')
+@logged_in
+def test_schedule_advisory(db, client):
+    resp = client.post(url_for('schedule_advisory', avg=DEFAULT_GROUP_NAME), follow_redirects=True, data={'advisory_type': issue_types[1]})
```
Comment (SantiagoTorres) [inline test/test_advisory.py]: we probably want to add an assert_redirects on these tests too.

File: test/test_advisory.py [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+    assert 1 == advisory_count()
+
+
+@create_package(name='foo', version='1.2.3-4')
+@create_group(id=DEFAULT_GROUP_ID, packages=['foo'], affected='1.2.3-3', fixed='1.2.3-4')
+@create_advisory(id=DEFAULT_ADVISORY_ID, group_package_id=DEFAULT_GROUP_ID, advisory_type=issue_types[1])
+@logged_in
+def test_edit_advisory(db, client):
+    workaround = 'the cake is a lie'
+    impact = 'Big shit and deep trouble!'
```
Comment (SantiagoTorres) [inline test/test_advisory.py]: 💯

**Comment by anthraxx** [comment]
- replaced the .rjust()

**Pull request merged**